src/routers/sessions.py

```
@@ -632,19 +675,17 @@ async def get_session_peers(
-    ],
 )
 async def get_session_context(
     workspace_id: str = Path(...),
     session_id: str = Path(...),
+    jwt_params: JWTParams = Depends(
+        require_auth(
+            workspace_name="workspace_id",
+            session_name="session_id",
+            allow_member_read=True,
+        )
+    ),
```

Ruff (0.16.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)